packages/strapi-design-system/src/Carousel/Carousel.js

```diff
@@ -1,33 +1,8 @@
-import React, { Children, cloneElement, useRef } from 'react';
+import React from 'react';
```

Comment:
IMO this component should be named CarouselInput as it uses the Field components. The other one is a simple Carousel